ui/shared/AssetSelector.qml

```diff
@@ -8,8 +8,9 @@ Item {
     id: root
     property var assets
     property var selectedAsset: {
-        "symbol": "snt", "name": "", "value": "", "fiatValue": "", "address": ""
+        "symbol": "snt", "name": "", "value": "", "fiatValue": "", "address": "", "fiatBalance": ""
     }
+    signal assetSelected(var selectedAsset)
```

Comment:
@emizzle lemme know if anything speaks against this signal

Reply:
Not at all, I was thinking to add it, but didn't see an immediate need at the time. Well done!

Reply:
In thinking about this again, would we be able to use the built-in `onSelectedAssetChanged` instead?